### DISCUSSION TITLE
how to build an smtp

### DISCUSSION THREAD

**Discussion by Uganto** [community/community#50145] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

How to build an smtp

**Comment by ettaboyle** [comment]
Hi @Uganto,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on smtp. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic. We've closed this topic and encourage you to check out the [GitHub Community Code of Conduct ](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct)as you get started as a contributor.

Best of luck!
